src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -177,11 +172,19 @@
   private WebElement footer;
   @FindBy(css = "#globalNavigation")
   private WebElement globalNavigationBar;
+
   private String globalNavigationAvatarPlaceholder = ".avatar-container.logged-avatar-placeholder";
-  private String loggedInUserSelectorMonobook = "#pt-userpage a[href*=%userName%]";
-  private String loggedInUserSelectorMercury = ".avatar img[alt*=%userName%]";
   private VenusGlobalNavPageObject venusGlobalNav;
 
+  protected final static By LOGIN_BUTTON_CSS = By.cssSelector("a[data-id='login']");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)